country-dashboard/src/index.css

```diff
@@ -1,13 +1,65 @@
-body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+*,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p
+ {
+  margin: 0px;
+  padding: 0px;
+  box-sizing: border-box;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
-    monospace;
+html {
+  max-width: 1280px;
+  margin: auto;
 }
+body
+{
+ max-width:1100px;
+ margin: auto;
+}
+
+.App {
+  display: flex;
+  flex-direction: column;
+  gap: 50px;
+}
+
+.header {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  padding-left: 50px;
+}
+.header-Search
+{
+  display: flex;
+  gap: 20px;
+
+}
+.main-Content {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  gap: 20px;
+}
+.card--ui {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  gap: 10px;
+  border: 1px solid black;
+  padding: 10px;
+}
+.card-text {
+  display: flex;
+  flex-direction: column;
+  gap: 5px;
+}
+.card-flag img {
```

Comment:
Don't use px and don't hardcode image width or height. That is why we can see the flag's aspect ratio is quite off
<img width="234" alt="image" src="https://github.com/rumoravenue/rumor-react-training-a2/assets/62576642/8f698022-c295-4ad2-a0df-781b1f02e5e8">

-3

Reply:
Hay , Shrinjay 
on the PX we have the discussion in the call and you  told that , form now the project part which you have submitted don't change it , take care in future ......

Reply:
Ok thanks....